x-pack/solutions/security/packages/kbn-securitysolution-exceptions-common/docs/openapi/ess/security_solution_exceptions_api_2023_10_31.bundled.schema.yaml

```diff
@@ -3259,8 +3259,23 @@ components:
       example: exception-list.attributes.name:%Detection%20List
       type: string
     ListId:
```

Comment:
This change would make sense here, in that we're pointing to the value list `id` and `type` when adding an exception item that references a value list.